esp-hal/src/sdio/result.rs

```diff
@@ -0,0 +1,39 @@
+//! Error and result types for SDIO peripherals.
+
+/// Convenience alias for the SDIO peripheral [Result](core::result::Result) type.
+pub type Result<T> = core::result::Result<T, Error>;
```

Comment:
I don't think this is something that we want, if you need it for the eh impl then please inline it close to where the eh impl will live

Reply:
I removed the `Result` alias, but did keep the `Error` type. `Error` currently only holds one variant, but the plan is to implement more for specific failure conditions for the `hal::mmc` implementation.